js/views/stock.js

```diff
@@ -1 +1,53 @@
 // define your StockView here
+var StockView = function( selector, model ){
+  this.el = document.querySelector( selector );
+  this.model = model;
+  this.render();
+  this.lookupSymbol();
+  this.updateShares();
+}
+
+// Stock prototype
+StockView.prototype = {
+  
+  render: function() {
+    var companyName = document.querySelector(".company-name");
+    companyName.innerHTML = stock.companyName;
+    var price = document.querySelector(".price");
+    price.innerHTML = "$" + stock.price.toFixed(2);
+    var totalValue = document.querySelector(".total-value");
+    totalValue.innerHTML = "The total value at " + stock.numShares + " shares is $" + stock.totalValue + "."; 
+  },
+
+  lookupSymbol: function() {
+    // When the user enters a symbol, instantiate a new instance of a stock model and associate it with the 'model' attribute of the view. Then re-render the view
+    var form = document.getElementById("#lookup-stock");
+    form.addEventListener("submit", onSubmit);
```

Comment:
Interesting you did it this way, with a brand-new private function. I'd just include it as an inline callback:
```
form.addEventListener("submit", function(){

});
```